Services/CO.CDP.Localization/StaticTextResource.cy.resx

```diff
@@ -1575,19 +1575,22 @@
     <value>Gall dau gyflenwr neu fwy sydd am gyflwyno cynnig ar y cyd greu consortiwm. Gallwch ychwanegu cynifer o gyflenwyr ag sydd ei angen.</value>
   </data>
   <data name="Consortium_ConsortiumStart_Paragraph2" xml:space="preserve">
-    <value>Fel y sefydliad sy'n creu'r consortiwm, bydd angen i chi roi: </value>
+    <value> </value>
   </data>
```

Comment:
This empty value isn't right. If we've not got a translation for this piece of text, the entire `<data>...</data>` element should be taken out